.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Sonar Analysis
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+jobs:
+  build:
+    name: Analyse Module with SonarCloud
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK 11
```

Comment:
This is not yet compatible with all projects.
It's still in progress.

Reply:
needed for sonar

Reply:
ok, so we need to wait on JDK 11 support for projects, isn't it ?

Reply:
we can activate the sonar analysis now. It is not blocker if fail on PR. And it will indicate us if something is not building on jdk 11.
I think we should not wait

Reply:
ok, let's do it :-)